README.es-ES.md

```
@@ -0,0 +1,131 @@
+
+
+<p align="center">
+  <a href="https://gitee.com/kyrzy0416/rconsole-plugin">
+    <img width="200" src="https://s2.loli.net/2024/08/19/ty5K6P3hsAaXC47.webp">
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)